server/http/server.go

```diff
@@ -44,27 +44,28 @@ func (s *Server) AddTopicEventHandler(topic, route string, handler func(ctx cont
 	}
 	s.topicSubscriptions = append(s.topicSubscriptions, sub)
 
-	s.mux.HandleFunc(route, func(w http.ResponseWriter, r *http.Request) {
-		w.Header().Add("Content-Type", "application/json")
-		content, err := ioutil.ReadAll(r.Body)
-		if err != nil {
-			w.WriteHeader(http.StatusBadRequest)
-			return
-		}
+	s.mux.Handle(route, optionsHandler(http.HandlerFunc(
+		func(w http.ResponseWriter, r *http.Request) {
+			w.Header().Add("Content-Type", "application/json")
```

Comment:
Please use predefined content-type header in net/http. `http.HeaderContentType` instead of string literal.

Reply:
net/http package does not define constants for content type (https://golang.org/pkg/net/http). Is this perhaps from diff http package?

Reply:
I have confused with fasthttp package. But I would define our own const instead of using string literal.